docker/README.md

```diff
@@ -0,0 +1,105 @@
+# [Devito] docker image library
+
+In order to facilitate the dissemination, usage, and development of Devito, we provide a series of docker images. These images support numerous architectures and compilers and are tagged accordingly. In the following, we describe the available images and the workflow to build it yourself. You can find all the available images at [DevitoHub](https://hub.docker.com/r/devitocodes/). 
+
+## [Devito] images
+
+Devito provides three main images that are targeting different architectures and/or using different compilers. In the following, all images are described as `imagename-*`. The `*` corresponds to, and should be swapped for, the different release tags `dev`, `latest` or `vx.x.x` depending on if you are interested in specific versions (`vx.x.x`), the latest stable release(`latest`), or the latest development status (`dev`)
+
+### [Devito] on CPU
+
+We provide two CPU images:
+- `devito:gcc-*` with the standard GNU gcc compiler.
+- `devito:icc-*` with the intel C compiler for intel architectures.
+
+these base images provide a working [Devito] environment for any CPU architecture and come with [Devito], `gcc/icc` and `mpi` preinstalled as well as utilities such as `jupyter` for usability and exploration of the package.
```

Comment:
These

Reply:
no need for square brackets surrounding Devito

Reply:
I'd shrink it as "[...] and come with [Devito]. Utilities such as `jupyter` ..."